packages/paste-website/src/component-examples/ChatComposerExamples.ts

```diff
@@ -0,0 +1,131 @@
+            )
+          );
+        },
+      }}
+    />
+  )
+}
+
+render(<RichTextExample/>)`.trim();
+
+export const MaxHeightExample = `const MaxHeightExample = () => {
+  return (
+    <ChatComposer
+      maxHeight="size10"
+      ariaLabel="A max height chat composer"
+      config={{
+        namespace: 'customer-chat',
+        onError (e) { throw e },
+        editorState () {
+          const root = $getRoot();
+
+          if (root.getFirstChild() !== null) return;
+
+          for (let i = 0; i < 10; i++) {
+            root.append(
+              $createParagraphNode().append(
+                $createTextNode('this is a really really long initial value')
+              )
+            );
+          }
+        },
+      }}
+    />
+  )
+}
+
+render(<MaxHeightExample />)`.trim();
+
+export const ChatDialogExample = `const ChatDialog = () => {
+  const scrollRef = React.createRef();
+
+  return <>
+    <Box overflowY="scroll" maxHeight="size50" tabIndex={0}>
+      <ChatLog>
+        <ChatBookend>
+          <ChatBookendItem>Today</ChatBookendItem>
+        </ChatBookend>
+        <ChatEvent>
+          <strong>Lauren Gardner</strong> has joined the chat ・ 4:26 PM
+        </ChatEvent>
+        <ChatMessage variant="inbound">
+          <ChatBubble>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</ChatBubble>
+          <ChatMessageMeta aria-label="said by Gibby Radki at 4:30pm">
+            <ChatMessageMetaItem>
+              <Avatar name="Gibby Radki" size="sizeIcon20" />
+              Gibby Radki ・ 4:30 PM
+            </ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatMessage variant="outbound">
+          <ChatBubble>Nulla sit amet elit mauris.</ChatBubble>
+          <ChatMessageMeta aria-label="said by you at 4:32pm">
+            <ChatMessageMetaItem>4:32 PM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+        <ChatMessage variant="inbound">
+          <ChatBubble>
+            Curabitur enim lorem, cursus et massa non, pretium faucibus lacus. Donec odio neque, consectetur a suscipit sit
+            amet, blandit id erat.
+          </ChatBubble>
+          <ChatMessageMeta aria-label="said by Gibby Radki at 4:48pm">
+            <ChatMessageMetaItem>4:48 PM</ChatMessageMetaItem>
+          </ChatMessageMeta>
+        </ChatMessage>
+      </ChatLog>
+    </Box>
+    <Box
+      ref={scrollRef}
+      borderStyle="solid"
+      borderWidth="borderWidth0"
+      borderTopWidth="borderWidth10"
+      borderColor="colorBorderWeak"
+      display="flex"
+      flexDirection="row"
+      columnGap="space30"
+      paddingX="space70"
+      paddingTop="space50"
+    >
+      <ChatComposer
+        config={{
+          namespace: 'foo',
+          onError: (error) => {
+            throw error;
+          },
+        }}
+        ariaLabel="Message"
+        placeholder="Chat text"
+      >
+        <AutoScrollPlugin scrollRef={scrollRef} />
```

Comment:
Suggestion: This example does show adding an additional plugin to the editor. Maybe you could just call it out on the page? If set a max height on the composer it will make more sense. This plugin makes it so the cursor is always in view as the chat composer gets taller.

Reply:
Or you could just add the autoscroll plugin to the max height example. Either works.